### ISSUE TITLE
TypeScript compiler exited with non-zero exit code

### ISSUE THREAD

**Issue by vtereshyn** [kamilkisiela/bob#147]
I am getting an error that `TypeScript compiler exited with non-zero exit code`. Meantime, `tsc -b` works well

Also, I noticed that `bob` doesn't respect my `tsconfig` compiler options. Maybe I am doing something wrong, but I was referring to `graphql-codegen` repo

BTW. Thank you for this tool. It looks amazing, especially `bob bootstrap` command.

**Comment by n1ru4l** [comment]
Hey @vtereshyn, I would love to help you, can you please provide more context? In the best case a failing PR or GitHub example. `bob` runs `tsc` and overwrites a few compiler options in order to generate correct esm and cjs code.

**Comment by vtereshyn** [comment]
UPD: The issue was in `This syntax requires an imported helper but module 'tslib' cannot be found.`. But for some reasons, this error was after `TypeScript compiler exited with non-zero exit code.`. 

When I went into `assertTypeScriptBuildResult` and put `if (result.exitCode !== 0 && result.code !== 0) {` instead of `if (result.exitCode !== 0) {` I was able to get error described above. Then, I installed `tslib` as a dependency, and now everything works well
